Find a functionally equivalent code:com.android.connectivitymanagertest.ConnectivityManagerTestBase.disconnectAP:COMMENT
<com.android.connectivitymanagertest.ConnectivityManagerTestBase: boolean disconnectAP()>
Method Modifier: public      protected   
Method Modifier: Yes         No          
Method Updated:  No          Yes         
Comment:/*
     * Disconnect from the current AP and remove configured networks.
     */

@@ -1,26 +1,28 @@
 {
     // remove saved networks
     if (!mWifiManager.isWifiEnabled()) {
-        log("Enabled wifi before remove configured networks");
+        logv("Enabled wifi before remove configured networks");
         mWifiManager.setWifiEnabled(true);
-        sleep(SHORT_TIMEOUT);
+        SystemClock.sleep(SHORT_TIMEOUT);
     }
     List<WifiConfiguration> wifiConfigList = mWifiManager.getConfiguredNetworks();
     if (wifiConfigList == null) {
-        log("no configuration list is null");
+        logv("no configuration list is null");
         return true;
     }
-    log("size of wifiConfigList: " + wifiConfigList.size());
+    logv("size of wifiConfigList: " + wifiConfigList.size());
     for (WifiConfiguration wifiConfig : wifiConfigList) {
-        log("remove wifi configuration: " + wifiConfig.networkId);
+        logv("remove wifi configuration: " + wifiConfig.networkId);
         int netId = wifiConfig.networkId;
         mWifiManager.forget(netId, new WifiManager.ActionListener() {
 
+            @Override
             public void onSuccess() {
             }
 
+            @Override
             public void onFailure(int reason) {
-                log("Failed to forget " + reason);
+                logv("Failed to forget " + reason);
             }
         });
     }

